src/main/java/duke/Main.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import duke.ui.DialogBox;
+import duke.ui.MainWindow;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private final Duke duke = new Duke("src/main/data/duke.txt");
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
Perhaps create a header comment for this method (since it is a public method)?

Reply:
Yep! Thanks for the reminder!